### ISSUE TITLE
Dot notation in headers key

### ISSUE THREAD

**Issue by silentlight** [abdennour/react-csv#84]
I try to add my headers like so:

```
headers = [
        { label: 'Id', key: 'id' },
        { label: 'First Name', key: 'profile.firstName' },
        { label: 'Last Name', key: 'profile.lastName' },
        { label: 'Email', key: 'email' },
        { label: 'Phone number', key: 'profile.phoneNumber' },
        { label: 'Created at', key: 'createdAt' },
      ];
```

The problem is that first name, last name and phone number are empty. Doesn't react-csv support dot notation for `key`?

Thanks

**Comment by mccabemj** [comment]
@silentlight seems like it is not supported. Looking at the source, having something like `'profile.firstName'` ends up compiling to `object['profile.firstName']` which is undefined obviously so defaults to blank string. I'll do a PR today to add support for this

**Comment by mccabemj** [comment]
I have submitted a PR to add this feature #107

**Comment by abdennour** [comment]
If someone is interested in being a contributor in this project, please let [contact me](http://in.abdennoor.com).

While it's hard to review all issues/PR due to time limitation, it is also not acceptable to approve PRs without review.

**Comment by mccabemj** [comment]
Merged this feature in [v1.0.18](https://github.com/abdennour/react-csv/releases/tag/1.0.18)
